src/build.mjs

```diff
@@ -101,5 +105,23 @@ try {
   console.error(e);
 }
 
+// If we have a global name, create a build for CDN.
+if (config.globalName) {
+  try {
+    await Promise.all([
+      // CDN build.
+      esbuild.build({
+        ...config,
+        entryNames: `[dir]/${config.globalName}`,
+        format: 'iife',
+        minify: true,
```

Comment:
Do we need to explicitly set `bundle: true` here to inline dependencies?
ESBuild API indicates it won't bundle by default.
https://esbuild.github.io/api/#bundle

Reply:
Good call!